src/puppet-web/wechaty-bro.js

```diff
@@ -378,6 +379,19 @@
     })
     return true
   }
+
+  function hookProcess() {
+    var chatFactory = WechatyBro.glue.chatFactory
+    var rootScope = WechatyBro.glue.rootScope
+    // hook chatFactory._recalledMsgProcess, resolve emit RECALLED type msg
+    chatFactory.__recalledMsgProcess = chatFactory._recalledMsgProcess
+    chatFactory._recalledMsgProcess = function (msg) {
+      var _this = this
```

Comment:
What's the reason that you save `this` to `_this`? 

IMHO, It seems not necessary to do that inside this function?